README.md

```diff
@@ -5,9 +5,10 @@ It uses [Hammr CLI](https://github.com/usharesoft/hammr) to create, generate and
 
 ## Prerequisite
 - The plugin requires at least Jenkins version 2.150.2 to work.
-- `pip`,`virtualenv` and `python2.7` need to be installed on the executor.
+- `python2.7` needs to be installed on the executor.
 - You also need a running UForge AppCenter server (minimum version: 3.8.13).
 
+If the "Installation Step" of the plugin fails, you may need to install some more dependencies on your Jenkins executor. See: http://docs.usharesoft.com/projects/hammr/en/latest/pages/getting-started/installation.html#installing-hammr for more information.
```

Comment:
I know I suggested this sentence but I'm wondering if it's a good idea to link hammr documentation with "install python" step...